### ISSUE TITLE
Web -  Step 6 - Topic 2 - Preventing Recurrence - Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#75]
The SVG left, and right-pointing arrow is missing a title, description, or some piece of ARIA as part of the Home Page and other buttons/links. Same applies to the bookmarks icon. There is no ALT text or some kind of description for the SVG image of the light bulb that is decorative before the "Example in Action: Responsible Sourcing Tool 1A" text or the left and right pointing arrows for the Previous and Next Step links. They should at least have alt="" if nothing else. There are four other images most likely in the footer that I couldn’t see without having my tools in IE to find that are empty or missing ALT text.

There might be missing ALT text on other pages that I didn't have time to check so please those too.

Step 6 - Topic 2 - Preventing Recurrence - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/steps/6/topic/2

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we address the lightbulb alt text?
John, thanks for looking into this. However, I do not see a left or right pointing arrow (except for the arrow to HOMEPAGE that we are currently addressing. Can you please point out where the right facing and left facing arrows are? Also, can you please let us know about the missing images? I cannot see them and am unable to address the issue unless I can specify them for our developers.

Thanks!
Kyle

**Comment by jfc3-dol** [comment]
The left and right arrows are in the "Previous" and "Next" buttons which will be displayed on all the pages that use them not only this page.

Please see attached screenshot.

![comply chain - web - left and right arrows in previous and next buttons](https://user-images.githubusercontent.com/10423487/61249941-754aac00-a724-11e9-8305-c4f6b211b169.png)

**Comment by Abychalil** [comment]
Alt texts provided for light bulb, Home, Previous, Next buttons..

**Comment by jfc3-dol** [comment]
There is now ALT text on the left pointing arrow and other items, so I'm going to close the issue.
